docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+title: Action Options
+permalink: /documentation/test-api/actions/action-options.html
+---
+# Action Options
+
+This topic describes two types of [test action](index.md) options.
```

Comment:
There some mistakes here. We have three types of options:
`MouseOptions`, `ClickOptions`, `TypeOptions`.
```js
'MouseOptions': {
    offsetX,
    offsetY,
    modifiers: { ctrl, alt, shift, meta }
}

'ClickOptions': MouseOptions + { caretPos }
'TypeOptions': ClickOptions + { replace }
```

As for actions:
```js
click, rightClick, doubleClick: ClickOptions
typeText: TypeOptions
others (drags, hover): MouseOptions
```